portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
-#content {
+body {
+    margin: 0px;
+    background-color: 
+}
+
+#title {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
+#left-column {
   margin-top: 20px;
 }
+
+#right-column {
```

Comment:
These id selectors refer to ids that don't exist in the document yet. Let's add them when they're used.